app/templates/static/contact-us.html

```diff
@@ -39,7 +40,7 @@ <h2 class="question__title neptune">
                                 Telephone
                             </h2>
                             <div class="question__description mars">
-                                <p>0300 1234 931</p>
+                                <p>{{helpers.telephone_number('')}}</p>
```

Comment:
If you drop the `''` so it's just `helpers.telephone_number()` it will prevent an empty 'aria-describedby` element.

Reply:
OK, will do. Wasn't entirely sure what to do in the case where there wasn't an existing element for this